src/containers/JobList/JobList.tsx

```diff
@@ -1,26 +1,20 @@
 import { useEffect } from "react";
-import { useDispatch, useSelector } from "react-redux";
+import { useDispatch } from "react-redux";
 import { fetchJobs } from "../redux/actions/jobActions";
-
+import JobComponent from "../JobComponent/JobComponent";
```

Comment:
Please rename file `JobComponent.tsx` to `index.tsx`. It will make this shorter. Change all of the other places